<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(123, 175, 212);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="print">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>31</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
	 background-color: rgb(210, 210, 210);
}

QPushButton:focus {
	background-color: rgb(210, 210, 210);
}

QPushButton {
	background-color: rgb(193, 193, 193);
}</string>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="home">
      <property name="minimumSize">
       <size>
        <width>161</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>31</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
	 background-color: rgb(210, 210, 210);
}

QPushButton:focus {
	background-color: rgb(210, 210, 210);
}

QPushButton {
	background-color: rgb(193, 193, 193);
}</string>
      </property>
      <property name="text">
       <string> Return to Main Menu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;From:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="fromDate">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* the box that is displayed*/
QDateEdit {
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px
}
/*toolbar on top of calendar*/
QCalendarWidget QToolButton {
height: 30px;
width: 75px;
color: white;
font-size: 12px;
icon-size: 24px, 24px;
font-weight: bold;
background-color: rgb(123,175,212);
}
QCalendarWidget QMenu {
width: 85px;
left: 10px;
color: white;
font-size: 12px;
background-color: rgb(123, 175, 212);
}
QCalendarWidget QSpinBox {
width: 75px;
font-size:12px;
color: white;
background-color: rgb(123, 175, 212);
selection-background-color: rgb(136, 136, 136);
selection-color: rgb(255, 255, 255);
}
QCalendarWidget QSpinBox::up-button { subcontrol-origin: border; subcontrol-position: top right; width:35px; }
QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right; width:35px;}
QCalendarWidget QSpinBox::up-arrow { width:28px; height:28px; }
QCalendarWidget QSpinBox::down-arrow { width:28px; height:28px; }

/* header row */
QCalendarWidget QWidget { 
alternate-background-color: rgb(240, 240, 240); 
}
/* normal days */
QCalendarWidget QAbstractItemView:enabled
{
font-size:12px;
color: black;
background-color: white;
selection-background-color: rgb(123, 175, 212);
selection-color: white;
}
/* days in other months */
/* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{
background-color: rgb(123, 175, 212);
}
QCalendarWidget QAbstractItemView:disabled
{
color: rgb(180,180,180);
}</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;To:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="toDate">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>22</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* the box that is displayed*/
QDateEdit {
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px
}
/*toolbar on top of calendar*/
QCalendarWidget QToolButton {
height: 30px;
width: 75px;
color: white;
font-size: 12px;
icon-size: 24px, 24px;
font-weight: bold;
background-color: rgb(123,175,212);
}
QCalendarWidget QMenu {
width: 85px;
left: 10px;
color: white;
font-size: 12px;
background-color: rgb(123, 175, 212);
}
QCalendarWidget QSpinBox {
width: 75px;
font-size:12px;
color: white;
background-color: rgb(123, 175, 212);
selection-background-color: rgb(136, 136, 136);
selection-color: rgb(255, 255, 255);
}
QCalendarWidget QSpinBox::up-button { subcontrol-origin: border; subcontrol-position: top right; width:35px; }
QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right; width:35px;}
QCalendarWidget QSpinBox::up-arrow { width:28px; height:28px; }
QCalendarWidget QSpinBox::down-arrow { width:28px; height:28px; }

/* header row */
QCalendarWidget QWidget { 
alternate-background-color: rgb(240, 240, 240); 
}
/* normal days */
QCalendarWidget QAbstractItemView:enabled
{
font-size:12px;
color: black;
background-color: white;
selection-background-color: rgb(123, 175, 212);
selection-color: white;
}
/* days in other months */
/* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{
background-color: rgb(123, 175, 212);
}
QCalendarWidget QAbstractItemView:disabled
{
color: rgb(180,180,180);
}</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="find">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover {
	 background-color: rgb(210, 210, 210);
}

QPushButton:focus {
	background-color: rgb(210, 210, 210);
}

QPushButton {
	background-color: rgb(193, 193, 193);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTableWidget" name="qaReportTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(240, 240, 240);
border-style: solid;
border-width: 1px</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Sample ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Clinician</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tech Reported</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date Reported</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Days in Culture</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
